### PR TITLE
Create fibonacci_sequence_DNC.py

### PR DESCRIPTION
**Pull request by easy-brother**
Find Fibonacci sequence quickly by divide and conquer

### CODE REVIEW

File: divide_and_conquer/fibonacci_sequence_DNC.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	 |                  |   =   |       |
+	 | F(n)   ,  F(n-1) |       | 1 , 0 |
+
+	 We can therefore compute the (n)th power of A = ((1,1),(1,0)) using recursive squaring
+
+	 Time complexity: O(log(n)), because it is a divide and conquer algorithm
+"""
+
+# return (n)th term of Fibonacci sequence
+def getFibonacci(n):
```
Comment (onlinejudge95) [inline divide_and_conquer/fibonacci_sequence_DNC.py]: Please have a look at CONTRIBUTING.md
We expect `method` and `variable` names to be in **snake_case**

Comment (onlinejudge95) [inline divide_and_conquer/fibonacci_sequence_DNC.py]: Also please write `doctests` for all your **methods/functions**

**Comment by onlinejudge95** [comment]
https://github.com/TheAlgorithms/Python/blob/master/Travis_CI_tests_are_failing.md

**Comment by easy-brother** [comment]
I will modify it based on the comments you have left and Pull request again.

File: divide_and_conquer/fibonacci_sequence_DNC.py (hunk 1 above)
Comment (cclauss) [reply divide_and_conquer/fibonacci_sequence_DNC.py]: + type hints, doctests and https://travis-ci.com/TheAlgorithms/Python/builds/140784236

**Pull request closed without merging**